packages/SwingSet/src/controller/controller.js

```diff
@@ -387,6 +388,28 @@ export async function makeSwingsetController(
       return kpid;
     },
 
+    /**
+     * Queue a method call to an object represented by a kmarshal token
+     *
+     * @param {any} target
+     * @param {string|symbol} method
+     * @param {unknown[]} args
+     * @param {ResolutionPolicy} resultPolicy
+     */
+    queueToVatObject(target, method, args = [], resultPolicy = 'ignore') {
+      const targetKref = krefOf(target);
+      assert.typeof(targetKref, 'string');
+      if (method !== Symbol.asyncIterator) {
+        assert.typeof(method, 'string');
+      }
```

Comment:
Remotables can expose methods keyed by any built-in or registered symbol; we should just roll with that here IMO.
```suggestion
      typeof method === 'string' || typeof method === 'symbol' ||
        Fail`method must be a string or symbol, not ${q(typeof method)}`;
```